bindings/nodejs/lib/wallet/wallet.ts

```diff
@@ -155,27 +137,6 @@ export class Wallet {
         return this.methodHandler.getSecretManager();
     }
 
-    /**
-     * Generate an address without storing it.
-     */
-    async generateEd25519Address(
```

Comment:
Shouldn't we hold off removing this?

Reply:
That's a different generate address, this one here would just generate an address and return it, so exactly the same as the secret manager
The discussion is about the generate address inside of the wallet if you don't provide an address